src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -240,6 +242,28 @@ public void ConfigureServices(IServiceCollection services)
             );
 
             services.AddMergePolicies();
+
+
+            // Configure access to Azure App Configuration
+            string appConfigEndpointUri = Configuration["AppConfigurationUri"];
+            ConfigurationBuilder builder = new ConfigurationBuilder();
+            TokenCredential credential = appConfigEndpointUri.Contains("maestrolocal") ? 
+                new DefaultAzureCredential() : 
+                (TokenCredential) new ManagedIdentityCredential();
```

Comment:
Are we sure this will use the right identity when deployed?

Reply:
Spoke with Alex. I think you need to add the Maestro apps to the access list for the configuration store. Not dotnet-bot. Let me know if that doesn't make sense.

Reply:
Talked with @alexperovich . I added the identities that he said should be used.